src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Filters } from '../Filters/Filters';
+import { countActiveTodos } from '../../utils/countActiveTodos';
+import cn from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  setFilter: (filterBy: Filters) => void;
+  onDelete: (id: number) => void;
+  filter: Filters;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  filter,
+  setFilter,
+  onDelete,
+  todos,
+}) => {
+  const filterLinks = [
+    { key: Filters.All, href: '#/', label: 'All', dataCy: 'FilterLinkAll' },
+    {
+      key: Filters.Active,
+      href: '#/active',
+      label: 'Active',
+      dataCy: 'FilterLinkActive',
+    },
+    {
+      key: Filters.Completed,
+      href: '#/completed',
+      label: 'Completed',
+      dataCy: 'FilterLinkCompleted',
+    },
+  ];
```

Comment:
You can try to generate this array from the `Filters` enum with `Object.keys/values/entries`